src/main/scala/io/shiftleft/fuzzyc2cpg/passes/cfgcreation/CfgCreator.scala

```diff
@@ -0,0 +1,404 @@
+package io.shiftleft.fuzzyc2cpg.passes.cfgcreation
+
+import io.shiftleft.codepropertygraph.generated.{EdgeTypes, Operators, nodes}
+import io.shiftleft.codepropertygraph.generated.nodes.CfgNode
+import io.shiftleft.fuzzyc2cpg.passes.cfgcreation.Cfg.CfgEdgeType
+import io.shiftleft.passes.DiffGraph
+import io.shiftleft.semanticcpg.language._
+
+/**
+  * Translation of abstract syntax trees into control flow graphs
+  *
+  * The problem of translating an abstract syntax tree into a corresponding
+control flow graph can be formulated as a recursive problem in which
+  * sub trees of the syntax tree are translated and their corresponding
+control flow graphs are connected according to the control flow
```

Comment:
```suggestion
 * control flow graphs are connected according to the control flow
```
my bad, copy paste error on the suggestion